docs/configuration/mftf.md

```diff
@@ -15,6 +15,83 @@ SELENIUM_HOST=selenium-hub
+    extra_hosts:
+      - ${TRAEFIK_SUBDOMAIN:-app}.${TRAEFIK_DOMAIN}:${TRAEFIK_ADDRESS:-0.0.0.0}
+    environment:
+      - START_XVFB=false
+```
+
+Afterwards, you'll need to edit the MFTF `.env` configuration file (`dev/tests/acceptance/.env`) and specify the new `SELENIUM_HOST`:
+
+```
+SELENIUM_HOST=selenium-chrome
+```
+
+After doing so, you'll need to re-run the commands: `vendor/bin/mftf build:project` and `vendor/bin/mftf generate:tests`
+
+Some MFTF tests that test uploading images for Products (such as `AdminSimpleProductImagesTest` - runnable with: `vendor/bin/mftf run:test AdminSimpleProductImagesTest`) will fail using the default configuration, due to `nginx` not having a large enough `client_max_body_size`, or the `selenium-chrome` tab crashing because `selenium-chrome` has run out of memory.
+
+The `client_max_body_size` issue will present itself when attempting the `bmp` upload test in `AdminSimpleProductImagesTest`; it can be remedied by increasing `nginx`'s `client_max_body_size`.
+
+To do this, create the file `.warden/nginx/conf.d/custom.conf`, and populate it with:
+
+```
+client_max_body_size 100M;
+```
+
+Then, mount a volume to the `nginx` container by adding the following to `services:` in `.warden/warden-env.yml`:
+
+```
+  nginx:
+    hostname: nginx
+    volumes:
+      - ./.warden/nginx/conf.d/custom.conf:/etc/nginx/conf.d/custom.conf
+```
+
+Restart the environment with `warden env up -d`.
+
+The `selenium-chrome` crashing error can be detected by examining the logs (`warden env logs selenium-chrome`) for the following:
+
+```
+selenium-chrome_1  | 20:35:02.333 INFO - To upstream: {"using":"xpath","value":"\/\/div[contains(@class, \"loading-mask\")]"}
+selenium-chrome_1  | 20:35:02.387 INFO - To downstream: {"sessionId":"0dc75d43859e0c9d7a0e113827fb477b","status":13,"value":{"message":"unknown error: session deleted because of page crash\nfrom tab crashed\n  (Session info: chrome=64.0.3282.119)\n  (Driver info: chromedriver=2.35.528139 (47ead77cb35ad2a9a83248b292151462a66cd881),platform=Linux 4.19.76-linuxkit x86_64)"}}
+selenium-chrome_1  | 20:35:02.388 INFO - Found handler: org.openqa.selenium.remote.server.commandhandler.GetLogsOfType@1345b27e
+selenium-chrome_1  | 20:35:02.388 INFO - /session/0dc75d43859e0c9d7a0e113827fb477b/log: Executing POST on /session/0dc75d43859e0c9d7a0e113827fb477b/log (handler: GetLogsOfType)
+selenium-chrome_1  | 20:35:02.389 INFO - To upstream: {"type":"browser"}
+selenium-chrome_1  | 20:35:02.392 INFO - To downstream: {"sessionId":"","status":6,"value":{"message":"no such session\n  (Driver info: chromedriver=2.35.528139 (47ead77cb35ad2a9a83248b292151462a66cd881),platform=Linux 4.19.76-linuxkit x86_64)"}}
+```
+
+To remedy, the `shm_size` for the `selenium-chrome` container needs to be increased from the default of 64M.
+
+Edit `.warden/warden-env.yml`, and `shm_size` as shown below:
+
+```
+version: "3.5"
+services:
+  selenium-chrome:
+    shm_size: '1gb'
```

Comment:
@Lunaetic @lbajsarowicz Is this shm_size fairly static or can it fluctuate? I.e. if we change the MFTM support to use chrome vs the spoke/hub (as it is currently) the shm_size should just be set in the defaults. If it's a fairly static need for this to be 1gb, then it can simply be used, otherwise it could be made a variable. Thoughts?

Reply:
1gb was an arbitrary size I chose while working on it; variable works, thanks

Reply:
Created a variable, set to 128mb, which still allows the tests to run from my testing.